src/components/SingleRev.jsx

```diff
@@ -199,15 +244,40 @@ export const SingleRev = () => {
           <br></br>
           {comments.length === 0 ? (
             <p>No Comments</p>
+          ) : deleteMsg ? (
```

Comment:
This doesn't persist, after deleting there is no message? Please review and fix before next PR